salt/modules/virt.py

```diff
@@ -1627,7 +1659,8 @@ def init(
         This is an optional parameter, all of the keys are optional within the dictionary. The structure of
         the dictionary is documented in :ref:`init-boot-def`. If a remote path is provided to kernel or initrd,
         salt will handle the downloading of the specified remote file and modify the XML accordingly.
-        To boot VM with UEFI, specify loader and nvram path.
+        To boot VM with UEFI, specify loader and nvram path or specify 'efi': ``True`` if your libvirtd version
+        is >= 5.4.0.
```

Comment:
Should we also add the minimal qemu version here? I have no idea which it is however

Reply:
Yeah, I will note 3.0.0 As the[ firmware metadata specification ] documented (https://github.com/qemu/qemu/blob/master/docs/interop/firmware.json)